src/components/Picture/index.jsx

```diff
@@ -2,5 +2,9 @@ import React from "react";
 import * as styles from "./Picture.module.scss";
 
 export default function Picture({ picture, sentence }) {
-  return <img className={styles.image} src={picture} alt={sentence} />;
+  return (
+    <div className={styles.image_container}>
+      <img className={styles.image} src={picture} alt={sentence} />
```

Comment:
We may want to think about using Gatsby's image handler. It can do wonderful things for performance rating. Though it does add some complications. I'll see about getting it implemented.